src/main/java/core/basesyntax/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.exception;
+
+public class UserNotFoundException extends RuntimeException {
+
+    public UserNotFoundException() {
+        super("User with given email doesn't exist.");
```

Comment:
it's hardcode, what if i wanna change the message 
use super(message) and pass message into the constructor